src/test/java/seedu/duke/commands/WithdrawCommandTest.java

```diff
@@ -70,27 +70,27 @@ public void processCommand_amountNotInt_shouldThrowException() {
     }
 
     @Test
-    public void getCurrency_validCurrency_shouldReturnCorrespondingCurrency(){
+    public void getCurrency_validCurrency_shouldReturnCorrespondingCurrency() {
         try {
             Method method = WithdrawCommand.class.getDeclaredMethod("getCurrency", String.class);
             method.setAccessible(true);
             WithdrawCommand command = new WithdrawCommand("add JPY 200");
```

Comment:
I think the command input here should be "withdraw" instead of "add", which also applies for some of the other tests